code/obj/item/mob_parts/robot_parts.dm

```diff
@@ -474,6 +482,7 @@ ABSTRACT_TYPE(/obj/item/parts/robot_parts/chest)
 /obj/item/parts/robot_parts/chest/standard
 	name = "standard cyborg chest"
 	desc = "The centerpiece of any cyborg. It wouldn't get very far without it."
+	material_amt = 1.2
```

Comment:
these magic numbers are gross, they should be based off of the other values

unfortunately, the only really good way to do that at compiletime isn't until 515

you could do it on the parent i guess, it's not like setting the var is /that/ expensive

Reply:
I am not quite sure what you are suggesting me to do. I could use defines and do `material_amt = ROBOT_PART_LIMB_COST * ROBOT_PART_LIGHT_COST_MODIFIER` that way they could also be used on the fabricator recipes